### DISCUSSION TITLE
Call for AOSC OS Wallpapers (2023 H1)

### DISCUSSION THREAD

**Discussion by MingcongBai** [AOSC-Dev/aosc-os-abbs#4474] [General]
In the past few years, thanks to active participation from our community members, our wallpaper collection has grown substantially (currently containing more than 100 wallpapers). Now that it has been over six months since our last call for wallpapers, we welcome your submissions!

The volume of submissions last year had been vast and the size of our wallpaper package exponentially. In an attempt of controlling its size, the contributors will vote for ten (10) candidates for inclusion in the default wallpaper collection, these candidates will then go into a community poll to produce our next default wallpaper. The rest of the submissions will go into another non-preinstalled package in the interest of controlling our distribution size. 

## Rules

1. **You must be the original author of your submission**, though you may submit photographs or graphic designs.
2. Your submission(s) must be of **3000x2000 or higher resolution**, unless they are used as tiled patterns.
3. Your submission **must not contain trademarks or (virtual or real-life) human(oid) portraits**, except for An-An and Tong-Tong.
4. Your submission **must not contain political, racial, or religious opinions.**

## Submission Format

1. Upload your submission in a reply below.
2. Note the following under **each of your submissions:** title of wallpaper, your name and e-mail, and licensing terms.

*Note: The rules and expectations above are subject to interpretation by our community contributors. After the submission deadline, community contributors will convene to audit and filter all submissions based on rule compliance and quality. We will hold an internal vote to select ten candidates for the community poll.*

**Submissions will close at 0:00 on April 9th, UTC +8 time.**

Thank you in advance for your contribution!

---

多年来，得益于来自社区的踊跃投稿，AOSC OS 的壁纸包内容愈发丰富（目前包含超过 100 张）。距离上次壁纸征集已有半年，现征集社区壁纸投稿，欢迎大家参与！

考虑到去年投稿量非常大，导致壁纸包大小急剧增长，今年我们将在贡献者间进行初步选拔，选出 10 张壁纸纳入默认的壁纸包中（其余投稿将包含在一个额外的壁纸包中，但考虑到对系统大小的影响，因此默认不预装）。今年上半年的默认壁纸将通过社区投票在通过初选的壁纸中选出。

## 投稿规则

— **投稿仅限原创作品**，但格式不限（摄影作品或平面设计均可）
— 除非作为平铺图案使用，壁纸分辨率**必须高于 3000x2000 分辨率**
— 投稿**不得包含商标及（虚拟及现实生活中）除安安/同同外的肖像**
— 投稿**不得包含政治、种族、宗教等观点性内容**

## 投稿流程

1. 直接跟帖回复要投稿的壁纸
2. 在 **每张** 壁纸链接下注明如下信息：壁纸标题、作者名称及邮箱地址，以及壁纸许可

**注：如上规则解释权归社区贡献者所有。在征集截止后，社区贡献者将召开例会，届时我们将根据合规性及质量考虑筛选壁纸投稿，并进行内部投票选出十张候选默认壁纸。**

**投稿截止时间：UTC +8 时间 4 月 9 日零时。**

感谢您的投稿！

**Comment by felixonmars** [comment]
投稿两张喵~

![IMG_20191009_120728.jpg](https://user-images.githubusercontent.com/1006477/229267500-e9ec8b6d-a7f7-477f-b6ba-521116802132.jpg)

Nyhavn i København

![2019-09-17-13-45-00.jpg](https://user-images.githubusercontent.com/1006477/229267565-d946cd41-43b9-48a8-b4f1-0c4379a7065b.jpg)

Suomen maantie

拍摄者：Felix Yan <felixonmars@archlinux.org>
授权：CC-BY

**Comment by cthbleachbit** [comment]
图片皆以 CC-BY-NC 发布，署名为 `cth451 <cth451@gmail.com>`。图像无默认说明以高度最大为 4320px 导出

### Brodhead Day+Night

![Brodhead in Daylight](https://static.cth451.me/DCIM/aosc2023/Brodhead%20in%20Blossom.jpg)
![Brodhead at Night](https://static.cth451.me/DCIM/aosc2023/Brodhead%20at%20Night.jpg)

### Chapel in Blossom

![Chapel in Blossom](https://static.cth451.me/DCIM/aosc2023/Chapel%20in%20Blossom.jpg)

### College & 4th St Crossing

![College & 4th St Crossing.jpg](https://static.cth451.me/DCIM/aosc2023/College%20%26%204th%20St%20Crossing.jpg)

### Fontana Access Bridge

![Fontana Access Bridge](https://static.cth451.me/DCIM/aosc2023/Fontana%20Access%20Bridge%20over%20Little%20Tennesee%20River.jpg)

### Lights in Botanical Garden

![Lights in Botanical Garden](https://static.cth451.me/DCIM/aosc2023/Lights%20in%20Botanical%20Garden.jpg)

### Northern Mocking Bird

![Northen Mocking Bird](https://static.cth451.me/DCIM/aosc2023/Northern%20Mocking%20Bird.jpg)

### Stuck in the Smokies

![Stuck in the Smokies](https://static.cth451.me/DCIM/aosc2023/Stuck%20in%20the%20Smokies.jpg)

### Sunset

![Sunset](https://static.cth451.me/DCIM/aosc2023/Sunset.jpg)

### Turkey Vulture

![Turkey Vulture](https://static.cth451.me/DCIM/aosc2023/Turkey%20Vulture.jpg)

### View from Moore's Knob

![View from Moore's Knob](https://static.cth451.me/DCIM/aosc2023/View%20from%20Moore%27s%20Knob.jpg)

**Comment by Nyovelt** [comment]
### UIUC ECE Building

![UIUC ECE Building](https://webresources.aaaab3n.moe/share/PXL_20220916_213642422.jpg)

拍摄者：Nicholas Wang [me@nicho1as.wang](mailto:me@nicho1as.wang)
授权：CC-BY-NC

### SD Desert

![SD Desert](https://webresources.aaaab3n.moe/share/_DSC4476.jpg)
https://webresources.aaaab3n.moe/share/_DSC4476.jpg

拍摄者：Alex Qin [canarypwn@aosc.io](mailto:canarypwn@aosc.io)
授权：CC-BY-NC

**Comment by M0gician** [comment]
## Embrace Me
![Embrace Me](https://user-images.githubusercontent.com/13779937/229417149-d0c74353-093b-4c8d-b376-6ff0a9765e70.jpg)

## Stair ⇨ Exit
![Stair ⇨ Exit](https://user-images.githubusercontent.com/13779937/229417663-1b496233-6b9b-429a-8576-4c90745c3adf.jpg)

## The Crest
![The Crest](https://user-images.githubusercontent.com/13779937/229417565-a8bafda5-7f78-4a9e-a4bb-4e2ef03c03a6.jpg)

## Dune of Weed
![Dune of Weed](https://user-images.githubusercontent.com/13779937/229418376-63a88f2e-9754-4f75-a378-996f1399d33d.jpg)

## Corporation Beach
![Corporation Beach](https://user-images.githubusercontent.com/13779937/229417818-d6b290ef-b8a3-43f7-b72f-3cf83eb78c0d.jpg)

## Corporation Beach II
![Corporation Beach II](https://user-images.githubusercontent.com/13779937/229417981-0ecdc3f9-c937-4e0f-948d-0952b814b072.jpg)

## Abandoned Sunset
![Abandoned Sunset](https://user-images.githubusercontent.com/13779937/229418163-c50dffda-0e39-438e-8aea-44760f571512.jpg)

## Artist's Bluff Lookout
![Artist's Bluff Lookout](https://user-images.githubusercontent.com/13779937/229418734-7004518d-ed9c-43b2-9b3d-da9828adbd7c.jpg)

## The Thrill Is Gone
![The Thrill Is Gone](https://user-images.githubusercontent.com/13779937/229418805-31e1cf9c-71be-4ac6-8832-14398b6cb3d3.jpg)

## Somewhere, Sometime
![Somewhere, Sometime](https://user-images.githubusercontent.com/13779937/229418930-ebdc7e0a-e4c6-4de2-9ea6-9c8234d5fc92.jpg)

拍摄者：Mogician Yang [m0g1c14n@gmail.com](mailto:m0g1c14n@gmail.com)
授权：CC-BY

**Comment by ImBearChild** [comment]
## Blurry Cat

![DSC01084](https://user-images.githubusercontent.com/29907700/229997350-66c38145-8760-40eb-9ca5-b0cd2d4a0612.JPG)

拍摄者： NianqingYao imbearchild@outlook.com
授权： [WTFPL](http://www.wtfpl.net/)

**Comment by George-Miao** [comment]
![image](https://user-images.githubusercontent.com/38549822/229999512-498852a7-3f09-4da0-85b1-a00717e0ca5a.jpeg)
Dawn

![image](https://user-images.githubusercontent.com/38549822/229999790-74ac2208-b276-43ec-80e5-e74517886c2d.jpeg)
Observatory, by George Miao <gm@miao.dev>, released under `CC BY-NC`

**Comment by nicholascw** [comment]
[Overcast](https://mirrors.nw.ci/public_bin/1.png)
[Indy](https://mirrors.nw.ci/public_bin/2.png)

Nicholas Wang [me@nicho1as.wang](mailto:me@nicho1as.wang)
授权：CC-BY-NC

**Comment by MingcongBai** [comment]
![Auspicious Clouds](https://user-images.githubusercontent.com/5006263/230582025-c6692dce-a7c7-441a-9b5a-d6f43940276b.jpg)

Auspicous Clouds / 祥云

![Castles of the Sea](https://user-images.githubusercontent.com/5006263/230582054-d0f2daf4-65c7-445d-98b6-dd319950414d.jpg)

Castles of the Sea / 海市

![Extraterrestrial](https://user-images.githubusercontent.com/5006263/230582134-26fa87c3-6e16-4c0d-85b2-0a349504595a.jpg)

Extraterrestrial / 异域

![Far Afield](https://user-images.githubusercontent.com/5006263/230582169-1f1e5ae1-d1e6-4f09-835f-15b62ef10586.jpg)

Far Afield / 向往

![Highway Blues](https://user-images.githubusercontent.com/5006263/230582218-527eb2bc-220e-44f9-945f-165587548766.jpg)

Highway Blues / 曲径

![Ice Plants](https://user-images.githubusercontent.com/5006263/230582260-54c51f43-3350-4ecd-8a6e-a6214286b823.jpg)

Ice Plants / 冰草

![Lux Adoris](https://user-images.githubusercontent.com/5006263/230582360-c408c589-bc30-43be-bf1c-5aa9f69843c0.jpg)

Lux Adoris / 炽热

![The Way Up](https://user-images.githubusercontent.com/5006263/230582399-c2de1eb0-10ea-4476-9a44-420d3708adae.jpg)

This Way Up / 登高

![Turf](https://user-images.githubusercontent.com/5006263/230582456-3a748d55-b943-4d2c-bbf7-c5b1f44b53d7.jpg)

Turf / 草丘

![Vista](https://user-images.githubusercontent.com/5006263/230582490-bb6b611a-7f7d-476d-81d3-008c9e83b3b1.jpg)

Vista / 远景

Mingcong Bai, jeffbai@aosc.io, CC-BY-NC

**Comment by Cyanoxygen** [comment]
![IMG_0306](https://user-images.githubusercontent.com/29174191/230722265-09b1c31a-0081-4da7-bbde-3e0b5bb99855.jpg)

Harmony

**Comment by MingcongBai** [comment]
Submissions closed. Thank you, everyone!
